packages/serverless-components/aws-cloudfront/lib/index.js

```diff
@@ -26,7 +26,7 @@ const createCloudFrontDistribution = async (cf, s3, inputs) => {
   const params = {
     DistributionConfig: {
       CallerReference: String(Date.now()),
-      Comment: inputs.comment,
+      Comment: inputs.comment ?? "",
```

Comment:
Per https://docs.aws.amazon.com/cloudfront/latest/APIReference/API_DistributionConfig.html comment is required and needs to be empty by default (if no comment is needed). Similar to update code to set to "" by default so that it can be deleted